src/client/pages/work/edit.jsx

```diff
@@ -36,7 +36,10 @@ const EditPage = ({ router }) => {
               thoughts
             } = data.workById;
             return (
-              <Mutation mutation={EDIT_WORK}>
+              <Mutation
+                mutation={EDIT_WORK}
+                refetchQueries={[{ query: GET_WORK_BY_ID, variables: { id } }]}
```

Comment:
@willjw3 You can probably run `refetchQueries` instead of using `cache-and-network` in the Query components.